src/utils/enums.py

```diff
@@ -0,0 +1,26 @@
+from foobar.exceptions import InvalidTransition
+
+
+def validate_transition(enum, from_state, to_state):
+    if not hasattr(enum, '_transitions'):
+        raise InvalidTransition('No transitions found')
+
+    transitions = enum._transitions.value
+    from_value = from_state.value if from_state is not None else None
+    if to_state.value not in transitions.get(from_value, []):
+        msg = 'Transition {0} -> {1} not allowed'.format(from_state, to_state)
+        raise InvalidTransition(msg)
+
+
+def get_direction_multiplier(enum, from_state, to_state, direction):
```

Comment:
This shouldn't really end up here as this function is specific for the wallet module.